src/main/java/com/box/sdk/BoxAPIRequest.java

```diff
@@ -58,7 +60,55 @@
     private int numRedirects;
     private boolean followRedirects = true;
     private boolean shouldAuthenticate;
-    private SSLContext sslContext;
+	//private static SSLContext sslContext;
+	private static SSLSocketFactory sslSocketFactory; 
```

Comment:
This line (and most of the others) are not indented the same as the rest of the file, which is causing the linter to fail.  Before we can merge the PR, all the lint errors will need to be fixed.  See https://travis-ci.org/box/box-java-sdk/jobs/427952040